mantis/cli/estimate_affine.py

```diff
@@ -72,35 +82,39 @@ def estimate_affine(source_position_dirpaths, target_position_dirpaths, output_f
     source_channel_Z, source_channel_Y, source_channel_X = source_channel_volume.shape[-3:]
     target_channel_Z, target_channel_Y, target_channel_X = target_channel_volume.shape[-3:]
 
-    focus_source_channel_idx = focus_from_transverse_band(
-        source_channel_volume[
-            :,
-            source_channel_Y // 2
-            - FOCUS_SLICE_ROI_WIDTH : source_channel_Y // 2
-            + FOCUS_SLICE_ROI_WIDTH,
-            source_channel_X // 2
-            - FOCUS_SLICE_ROI_WIDTH : source_channel_X // 2
-            + FOCUS_SLICE_ROI_WIDTH,
-        ],
-        NA_det=NA_DETECTION_SOURCE,
-        lambda_ill=WAVELENGTH_EMISSION_SOURCE_CHANNEL,
-        pixel_size=source_channel_voxel_size[-1],
-    )
+    if source_channel_Z < 2 or target_channel_Z < 2:
+        focus_source_channel_idx = 1
+        focus_target_channel_idx = 1
```

Comment:
This will break when registering 2D images with `shape[-3] = 1`. `focus_from_transverse_band` already returns `focus_idx = 0` for 2D images. What was the reason for making this change?

Reply:
It may also be good to rename these variables to `source_channel_focus_idx` and `target_channel_focus_idx`. For a minute I though this is a channel index rather than Z index